test/Microsoft.DotNet.Cli.Utils.Tests/BuiltInCommandTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public class BuiltInCommandTests : TestBase
+    {
+        [Fact]
+        public void TestExecute()
```

Comment:
Yes, this is a good idea.  Is it acceptable to put the expectation of the test in a `<summary>` comment on the test?  Or does it have to be in the name?